responsibleai/tests/causal/test_causal_general.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
+import shap
+from sklearn.model_selection import train_test_split
+
+from responsibleai import RAIInsights
+
+
+def get_adult_data():
+    X, y = shap.datasets.adult()
+    target_feature = "income"
+    y = [1 if y_i else 0 for y_i in y]
+
+    full_data = X.copy()
+    full_data[target_feature] = y
+
+    data_train, data_test = train_test_split(
+        full_data, test_size=1000, random_state=96132, stratify=full_data[target_feature]
+    )
+
+    categorical_columns = ["Race", "Sex", "Workclass",
+                           "Marital Status", "Country", "Occupation"]
+
+    return target_feature, categorical_columns, data_train, data_test
+
+
+def test_causal_classification_01():
```

Comment:
This test passes anyways so why do we need this test to run in our gates?

Reply:
Fair point, although I did think there was some value in showing the 'working' one. Have removed.